oshi-core/src/main/java/oshi/util/EdidUtil.java

```diff
@@ -32,6 +33,7 @@ private EdidUtil() {
      * @param edid The EDID byte array
      * @return The manufacturer ID
      */
+    @SuppressForbidden(reason = "using base 2")
```

Comment:
```suggestion
    @SuppressForbidden(reason = "customized base 2 parsing not in Util class")
```
I think we need to catch a NumberFormatException here?  Do you agree?

Reply:
if we're doing it in other similar places then, yes. Also its out of the scope of this pr I can open a new pr if u want to catch the exception